index.html

```diff
@@ -1,4 +1,5 @@
 <!DOCTYPE html>
+<<<<<<< HEAD
```

Comment:
This still has markings from git. I think you probably intended to delete these files?